### ISSUE TITLE
Are there any concrete plans to start interoperating or incorporating web-ext with Google Chrome, Opera, and other platforms?

### ISSUE THREAD

**Issue by tommai78101** [mozilla/web-ext#1130]
I understand the primary focus for web-ext is to improve and build upon its features for Mozilla Firefox. 

As noted here:

> By creating a new tool that focuses on the [emerging] web extension standard, we have a better chance of interoperating with other platforms, such as Google Chrome or Opera. It would be hard to do that while preserving compatibility in jpm.

I would like to know if there are any concrete plans to start interoperating with other platforms? Or it still needs to take a while before the main focus can be shifted towards this?

Thanks.

**Comment by rpl** [comment]
@tommai78101 yes, we have plan to add support for Chrome and Opera to web-ext run. 

Part of the refactoring have been already landed, with some additional tweaks that are going to be part of the support for Firefox for Android (#868), which uses the same underlying abstraction that we are going to be used to add the additional chrome/opera "extension runners".

**Comment by tommai78101** [comment]
Thank you. Reason I'm asking is because the quoted part isn't clear if there are plans to add Chrome and Opera support, and it's more like "this tool has potential to do XYZ."

**Comment by tgvdinesh** [comment]
Has this tool started supporting chrome?

**Comment by Pomax** [comment]
I guess same question still: is there a `--target=[chrome|edge|firefox]` on the near horizon that will not try to solve unsolvable differences, but automatically does `browser. => chrome.` substitutions and the like?

**Comment by kumar303** [comment]
> Is there a `--target=[chrome|edge|firefox]` on the near horizon

You can track https://github.com/mozilla/web-ext/issues/809 for Chrome support in web-ext

> ...automatically does browser. => chrome. substitutions...

This is probably out of scope for web-ext. I'd suggest using https://github.com/mozilla/webextension-polyfill which is well supported.

> ...automatically does ...manifest changes...

That one is a bit tricky. I'd suggest using something like webpack ([example](https://github.com/mdn/webextensions-examples/blob/master/react-es6-popup/webpack.config.js)) and configuring multiple targets for your output. Maybe there is a better strategy.

**Comment by matanster** [comment]
Seems to work with latest chrome releases ... at least for me.
